Circle.cpp

```diff
@@ -8,15 +8,19 @@
 
 #include "Shape.cpp"
 #include "Area.cpp"
+#include "Volume.cpp"
 #include "Input.cpp"
 
+
 using namespace std;
 class Circle : public Shape
 {
 
     private:
         double radius;
         double area;
+        double height;
```

Comment:
`area` and `volume` are derived properties while `radius` is a user input. So, it is better to group them as such.

Reply:
i didn't quiet understand your comment do you think we should declare area and volume in their own functions

Reply:
By derived I mean they are both calculated:

Area = PI * R^2
Volume = PI * R^2 * height.

Here R (radius) and height are the properties that are taken from the user, while area and height are calculated / derived. As such it is better to group them separately in your code to let the reviewer or whoever works on the code after you. This can be done like so:

    double area;
    double volume;

    //  Notice space here.
    double radius;
    double height;

In this arrangement of fields, it's immediately clear to the reader that there is something different about these fields inherently. Not to mention having spaces eases the burden on the eyes.

Reply:
corrected indentation https://github.com/Muhammadali2025222/C-Practice/pull/10 check for confirmation